lib/v2/components/custom_dialog.dart

```diff
@@ -0,0 +1,142 @@
+import 'package:flutter/material.dart';
+import 'package:seeds/v2/components/flat_button_long.dart';
+import 'package:seeds/constants/app_colors.dart';
+
+const double padding = 20;
+const double avatarRadius = 40;
+
+/// A custom dialog with top icon that can be used in multiple screens
+class CustomDialog extends StatelessWidget {
+  /// Top icon dialog
+  final Widget icon;
+
+  /// Dialog body content
+  final List<Widget> children;
+
+  /// Default title empty
+  final String leftButtonTitle;
+
+  /// Require define leftButtonTitle
+  final VoidCallback onLeftButtonPressed;
+
+  /// Default title empty
+  final String rightButtonTitle;
+
+  /// Require define rightButtonTitle
+  final VoidCallback onRightButtonPressed;
+
+  /// Default title empty
+  final String closeButtonTitle;
+
+  /// Default Navigator pop
+  final VoidCallback onCloseButtonPressed;
```

Comment:
this should not assume there will always be a close button 

What is one button is (Launch app) and the other button (Call number)

There is no close button action. 

Make it more generic please.

Reply:
oh i see, The close button is the big one. 

Still dont name it close. Name it something generic, like you with with the left button and right button